workflows/files/ZWEAMLCF.xml

```diff
@@ -0,0 +1,2151 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<workflow xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <autoTakeOwnership>true</autoTakeOwnership>
+    <!-- instance prefix settings -->
+    <workflowInfo>
+        <workflowID scope="none">config_apiml_zowe_30</workflowID>
+        <workflowDefaultName>Configuration of APIML for Zowe 3.0</workflowDefaultName>
+        <workflowDescription>Use this simplified workflow to configure APIML for Zowe 3.0</workflowDescription>
+        <workflowVersion>2</workflowVersion>
+        <vendor>Zowe</vendor>
+        <General/>
+    </workflowInfo>
+    <!--variables-->
+    <variable name="zowe_setup_dataset_prefix" scope="instance" visibility="public">
+        <label>Zowe setup MVS prefix</label>
+        <abstract>Where Zowe MVS data sets will be installed</abstract>
+        <description>Where Zowe MVS data sets will be installed</description>
+        <category>zowe</category>
+        <string valueMustBeChoice="false" multiLine="false">
+            <!-- Put validation here -->
+            <!-- Specify choices here -->
+            <default>IBMUSER.ZWEV2</default>
```

Comment:
All occurrences:
IBMUSER.ZWEV2 -> IBMUSER.ZWEV**3**

Reply:
Good catch for both. Those are already coming from V3, so I will fix that in Zowe V3. APIML workflow I have to rework based on Tuesday demo to APIML team, new use case updated, so I will close this pullRequest and open new one when I am ready. It will be slightly different (because of bit different use).